src/i18n/resources/fr/other.json

```diff
@@ -21,6 +21,7 @@
   "assessment": {
     "deskStudy": "Étude de bureau",
     "fra": "FRA",
+    "panEuropean": "paneuropéen",
```

Comment:
Does this apply to all languages? because I noticed some already have a translation for it. 
<img width="600" alt="image" src="https://github.com/user-attachments/assets/58e05189-780c-42e0-aeec-acdb35f3b2dc">
I can edit those if needed.